File: Lista 01/Python/questao_1_c_NovoSalario.py
#Receber o salário de um funcionário
#e calcular o reajuste sabendo do ganho de 4% de comissão de vendas

salario = float(input("Digite o valor do salário fixo: R$ "))
vendas  = float(input("Digite o valor total das vendas: R$ "))

salario = salario + (vendas * 0.04)

print("Salário final vale: R$", salario)

File: Lista 01/C/questao_1_c_NovoSalario.c
#include <stdio.h>

int main() {

	float salario, vendas;

	//Recebe o valor do salário
	printf("Digite o valor do salário: R$ ");
	scanf("%f",&salario);

	//Recebe o valor das vendas
	printf("Digite o valor das vendas: R$ ");
	scanf("%f",&vendas);

	//Calcula o reajuste do salário
	salario = (salario + (vendas * 0.04));

	//Mostra o valor reajustado
	printf("O valor do salário reajustado vale: R$ %.2f",salario);

	return 0;
}